### DISCUSSION TITLE
MkDocs team call.

### DISCUSSION THREAD

**Discussion by lovelydinosaur** [mkdocs/mkdocs#3671] [General]
Scheduling an open call this Saturday to discuss maintainership, funding, and project direction...
If you're interested, please indicate a preferred time (or times).

https://doodle.com/meeting/participate/id/bo6jLDja

I'll send out a meeting link in due course.

Will also plan on updating this discussion after the call.

  &mdash; Tom 💜

**Comment by Andre601** [comment]
What options of communication would be there? Only VC? Text chat? Or just listening in?

**Reply by lovelydinosaur** [reply]
Was planning on a video call.

**Reply by Andre601** [reply]
I personally could only listen in at most and maybe have text-messages, if available.

**Reply by lovelydinosaur** [reply]
I don't think that's going to work for me @Andre601, no. Seems like an odd request?
If we're going to have a properly functioning maintenance team, I'd like to have some more personable idea of who we all are.

**Reply by Andre601** [reply]
Always remember that not everyone has the hardware (i.e headset) necessary and that such a rewuirement is quite limiting.

There can be people that could be awesome members to the team, but requirements like that prevent them from properly participate.

**Reply by lovelydinosaur** [reply]
Let's see... I'll post link in due course to anyone interested and we can figure out what works as we go.

**Reply by Andre601** [reply]
Also, another point is, that a live-chat of sorts during this call could be beneficial for community inclusion, because they could ask questions for a fast reply instead of asking in this discussion and having to wait...

**Reply by pawamoy** [reply]
I suppose what is needed here is a clear explanation of what purpose this call serves: quickly exchanging/synchronizing between core team members, or engaging with the community at large on the mentioned topics. Could let us know what you had in mind @tomchristie :slightly_smiling_face:?

**Reply by lovelydinosaur** [reply]
> Could let us know what you had in mind @tomchristie 🙂?

Sure thing, yup... https://github.com/mkdocs/mkdocs/discussions/3677

**Reply by lovelydinosaur** [reply]
https://github.com/mkdocs/mkdocs/discussions/3677#discussioncomment-9239499

**Comment by lovelydinosaur** [comment]
@mkdocs/community @mkdocs/core - I'd like to setup a team call to talk about the future of MkDocs. I've got some work of my own to show, plus whatever any of y'all would like to bring to the table.

Some time options to choose from here... https://doodle.com/meeting/participate/id/e3K7A69e

**Reply by squidfunk** [reply]
Excited to see what you came up with and your vision for MkDocs ☺️ @alexvoss can you join in as well?

**Reply by alexvoss** [reply]
Yup, sounds good. Looking forward to it.

**Reply by lovelydinosaur** [reply]
@squidfunk @alexvoss @humitos @pawamoy Okay looks like we've got Thurs 1st 2:00-3:00 (GMT+1).

Let's see each other here... https://meet.google.com/jcx-vbcg-ujw

**Reply by alexvoss** [reply]
I suspect the Doodle UI has struck again so @pawamoy needs to be tagged, too. See you later today.

**Reply by alexvoss** [reply]
Thanks to all who attended the meeting. Here are my notes:

Tom showed a demo of the [sketch](https://github.com/mkdocs/sketch) for a future MkDocs and emphasizes that this is (still?) exploratory work.

Headline features are:

- Pages are built on-the-fly when requested, no "[build the whole site](https://github.com/mkdocs/sketch?tab=readme-ov-file#whats-the-big-deal)" step
- Focus on theming with a clean data model and context for templates
- Theming should not involve Python coding/code
- Extensibility via programmatic API

Concerns were raised about maintenance and migration efforts of plugins in the light of impending changes to the API. Tom stressed the need to first develop the new core and build the version that allows for safe rendering. A definition of an API and a migration path will then follow. 

Regarding maintenance, a request was made to elevate privileges of people currently helping with maintenance, for example so they can do releases. Tom says this is not a problem and suggests weekly team calls.

@tomchristie I hope I haven't forgotten anything noteworthy. Please feel free to add anything I missed!

**Reply by pawamoy** [reply]
Thank you @alexvoss for the notes, @tomchristie for sharing your progress, and @squidfunk and @humitos for sharing your perspectives too! Apologies for showing up late by the way :sweat:

**Reply by ultrabug** [reply]
Thanks for the notes @alexvoss!
